Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM python:3.8
 
-WORKDIR /
+ARG UID
+ARG GID
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
 
+WORKDIR /
 # install requirements
 COPY ./requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 
 # create a new user and use it.
-RUN useradd -M -u 1001 nonrootuser
-USER nonrootuser
+RUN mkdir -p /home/iceesuser
+RUN groupadd --system -g $GID iceesuser && useradd --system -g iceesuser --shell /bin/bash -u $UID --home /home/iceesuser iceesuser
```

Comment:
Is there a reason to add a system group here?

Reply:
Yes, otherwise, the group could have an id that is different from the user group id on the host, at least that was what happened on ebcr0. With group id added, it will make sure the container user has the same permission as the host user.